config.py

```diff
@@ -44,6 +44,11 @@ class Config(object):
     PARTY_PASSWORD = os.getenv('PARTY_PASSWORD')
     PARTY_AUTH = (PARTY_USERNAME, PARTY_PASSWORD)
 
+    REPORT_HOST = os.getenv('REPORT_HOST')
+    REPORT_PORT = os.getenv('REPORT_PORT')
+    REPORT_PROTOCOL = os.getenv('REPORT_PROTOCOL')
+    REPORT_URL = f'{REPORT_PROTOCOL}://{REPORT_HOST}:{REPORT_PORT}/'
```

Comment:
@JamesGardiner I was thinking a bit more along the lines of how it's done for SECURE_MESSAGE_URL